src/main/kotlin/ar/edu/unahur/obj2/vendedores/Vendedor.kt

```diff
@@ -28,6 +28,18 @@ abstract class Vendedor {
   fun otrasCertificaciones() = certificaciones.count { !it.esDeProducto }
 
   fun puntajeCertificaciones() = certificaciones.sumBy { c -> c.puntaje }
+
+  open fun influyente() =
+    false
```

Comment:
Para este ejercicio me parece mejor dejar la definición abstracta, lo que hiciste vos acá es poner una implementación por defecto. Lo que digo sería así:

```suggestion
  abstract fun influyente(): Boolean
```